apis/fluent-en_GB/atrium-api-fluent-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/fluent/en_GB/IterableContainsInOrderElementsOfAssertionSpec.kt

```diff
@@ -0,0 +1,37 @@
+package ch.tutteli.atrium.api.fluent.en_GB
+
+import ch.tutteli.atrium.creating.Expect
+import org.spekframework.spek2.Spek
+import kotlin.reflect.KFunction3
+
+class IterableContainsInOrderElementsOfAssertionSpec : Spek({
+    include(BuilderSpec)
+}) {
+    object BuilderSpec : ch.tutteli.atrium.specs.integration.IterableContainsInOrderOnlyEntriesAssertionsSpec  (
```

Comment:
my fault, should have been Values...
```suggestion
    object BuilderSpec : ch.tutteli.atrium.specs.integration.IterableContainsInOrderOnlyValuesAssertionsSpec  (
```